projects/vdk-plugins/vdk-trino/tests/test_vdk_trino_lineage.py

```diff
@@ -26,27 +28,225 @@
         VDK_TRINO_USE_SSL: "False",
     },
 )
-def test_lineage():
-    mock_lineage = mock.MagicMock(LineageLogger)
+def test_lineage_for_insert():
+
+    table_name = "test_table_" + uuid.uuid4().hex
+
+    mock_lineage_logger = mock.MagicMock(LineageLogger)
+
+    class TestConfigPlugin:
+        @hookimpl
+        def vdk_initialize(self, context):
+            context.state.set(LINEAGE_LOGGER_KEY, mock_lineage_logger)
+
+    runner = CliEntryBasedTestRunner(TestConfigPlugin(), trino_plugin)
+
+    result: Result = runner.invoke(
+        ["trino-query", "--query", f"create table {table_name} (test_column varchar)"]
+    )
+    cli_assert_equal(0, result)
+
+    insert_query = f"insert into {table_name} values('test value')"
+    result: Result = runner.invoke(["trino-query", "--query", insert_query])
+    cli_assert_equal(0, result)
+
+    # the lineage data is different on every run of the test,
+    # so we need this class to generalize the dict which is to be matched
+    class InsertLineageDataMatch:
+        def __eq__(self, lineage_data):
+            import json
+
+            print("INSERT query lineage data:")
```

Comment:
remove the prints?

Reply:
Was wondering about that. May be useful when debugging, but then again, you can run the tests locally and add the print. I will remove them.